strings/phobia.json

```diff
@@ -71,5 +71,87 @@
 		"anaconda",
 		"viper",
 		"python"
+	],
+
+	"robots": [
+		"robot",
+		"bot",
+		"borg",
+		"cyborg",
+		"android",
+		"ai",
+		"artificial intelligence",
+		"silicon",
+		"silly cone",
+		"beep",
+		"boop"
+	],
+
+	"doctors": [
+		"doc",
+		"md",
+		"viro",
+		"virologist",
+		"chem",
+		"chemist",
+		"geneticist",
+		"cmo",
+		"chief medical officer",
+		"nurse",
+		"medicine",
+		"healing",
+		"cryo",
+		"sleeper",
+		"chemistry",
+		"genetics",
+		"cloning",
+		"virology"
+	],
+
+	"authority": [
+		"cap",
+		"captain",
+		"chief",
+		"head",
+		"hop",
+		"hos",
+		"ce",
+		"cmo",
+		"rd",
+		"command",
+		"nanotrasen",
+		"admiral",
+		"centcom",
+		"comdom"
+	],
+
+	"the supernatural": [
+		"chaplain",
+		"chappy",
+		"chap",
```

Comment:
hey, old chap.

Reply:
Fiiine I'll remove that. Damn context-sensitivity...